api/client/client.go

```diff
@@ -0,0 +1,249 @@
+package client
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/base64"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/api/grpc/disperser"
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/Layr-Labs/eigenda/core/auth"
+	"github.com/Layr-Labs/eigenda/encoding"
+	"github.com/Layr-Labs/eigenda/encoding/utils/codec"
+	"github.com/ethereum/go-ethereum/log"
```

Comment:
I would say we should go with industry standard for rollup frameworks, but can safely ignore whatever decision EigenDA made since this component will live on the client side. OP uses Ethereum's logging library, I don't know about the others.